src/openai_summarizer.py

```
@@ -0,0 +1,147 @@
+class OpenAISummarizer:
+    def __init__(self):
+        """
+        Initializes the OpenAISummarizer instance.
+
+        Sets the initial number of API calls to 0 and configures the logging
+        to the DEBUG level.
+        """
+        self.calls = 0
+        logging.basicConfig(level=logging.INFO)
+
+    async def get_summary_from_ai(self, owner_of_messages, channel_messages, ai_prompts={}):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)